release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -733,6 +743,13 @@ The toolbox script is deprecated and support will be removed from a future {prod
 * `platform.vsphere.ingressVIP`
 * `platform.vsphere.network`
 
+[id="ocp-4-13-node-dep-topology-labels"]
+==== Kubernetes topology labels
+
+The commonly used Kubernetes topology labels `failure-domain.beta.kubernetes.io/zone` and `failure-domain.beta.kubernetes.io/region` have been replaced with `topology.kubernetes.io/zone` and `topology.kubernetes.io/region`, respectively. The replacement labels are available starting with Kubernetes 1.17 and {product-title} version 4.4.
+
+Currently, both the deprecated and replacement labels are supported, but support for the deprecated labels is planned to be removed in a future release. To prepare for the removal, you can update your clusters to use the newer labels for any topology-based affinities that you have configured.
```

Comment:
What's the term that we generally use for the stuff customers add to their clusters? Do we call that workload? Cluster integrations? That's the stuff they need to update. The cluster itself sounds like the thing that we're on the hook to maintain and should be taking care of for them.

I'm fine with this as is, but if we have a commonly used term for that I think we should use that term rather than "update your clusters"

Reply:
how about something like: "modify any resources(e.g. volumes, deployments, other workloads) that reference the old labels to instead use the new labels"

?

Reply:
> What's the term that we generally use for the stuff customers add to their clusters?

Great question, and not something I end up writing about enough to know. I will solicit input from the larger docs team on that one :slightly_smiling_face:

Reply:
Seems like "workloads" is the consensus term here :+1:

Reply:
Hey Ben - sorry, I did not see you comment (probably because I did not refresh the page). Will revise with that input!

Reply:
revised - lmk it you think it works better now

Reply:
looks good to me.

Reply:
lgtm